R/geom-label.R

```diff
@@ -58,15 +62,17 @@ GeomLabel <- ggproto("GeomLabel", Geom,
   default_aes = aes(
     colour = "black", fill = "white", size = 3.88, angle = 0,
     hjust = 0.5, vjust = 0.5, alpha = NA, family = "", fontface = 1,
-    lineheight = 1.2
+    lineheight = 1.2, linetype = "solid", linewidth = 0.25
   ),
 
   draw_panel = function(self, data, panel_params, coord, parse = FALSE,
                         na.rm = FALSE,
                         label.padding = unit(0.25, "lines"),
                         label.r = unit(0.15, "lines"),
                         label.size = 0.25,
```

Comment:
If `label.size` becomes deprecated, it might be best to leave them out of the belly of `GeomLabel` as well.

Reply:
Update: I've fully deprecated `label.size`. It now only exists in `geom_label()` to give a deprecation warning.